--- .ansible/venv/site-packages/ansible/plugins/httpapi/restconf.py.bak	2020-03-10 00:59:46.043419363 +0300
+++ .ansible/venv/site-packages/ansible/plugins/httpapi/restconf.py	2020-03-10 00:57:24.190241825 +0300
@@ -69,7 +69,7 @@
 
 def handle_response(response, response_data):
     try:
-        response_data = json.loads(response_data.read())
+        response_data = json.loads(to_text(response_data.read()))
     except ValueError:
         response_data = response_data.read()
 
