--- .ansible/venv/site-packages/molecule/command/base.py.bak	2020-02-25 15:25:10.954976080 +0300
+++ .ansible/venv/site-packages/molecule/command/base.py	2020-02-25 15:24:04.358736109 +0300
@@ -38,7 +38,7 @@ from click_help_colors import HelpColors
 
 LOG = logger.get_logger(__name__)
 MOLECULE_GLOB = os.environ.get("MOLECULE_GLOB", "molecule/*/molecule.yml")
-MOLECULE_DEFAULT_SCENARIO_NAME = "default"
+MOLECULE_DEFAULT_SCENARIO_NAME = os.environ.get("MOLECULE_TARGET", "default")
 
 
 @six.add_metaclass(abc.ABCMeta)
